src/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.csproj

```diff
@@ -4,9 +4,9 @@
     <Configurations>netcoreapp-Debug;netcoreapp-Release;uap-Debug;uap-Release</Configurations>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="System.ComponentModel.cs" />
```

Comment:
This file wasn't really necessary. Moved its content to System.ComponentModel.TypeConverter.cs

Reply:
cc @maryamariyan